bootstrap/kubeadm/api/v1alpha2/zz_generated.conversion.go

```diff
@@ -289,14 +294,10 @@ func autoConvert_v1alpha3_KubeadmConfigSpec_To_v1alpha2_KubeadmConfigSpec(in *v1
 	out.Users = *(*[]User)(unsafe.Pointer(&in.Users))
 	out.NTP = (*NTP)(unsafe.Pointer(in.NTP))
 	out.Format = Format(in.Format)
+	// WARNING: in.Verbosity requires manual conversion: does not exist in peer-type
```

Comment:
Completely tangential, but I almost wish this threw an error with the fields that require manual conversion. That way we could more easily catch the need to handle additional fields for special handling in existing manual conversion methods. @ncdc Do you think this is something that the conversion-gen authors may be open to?